### DISCUSSION TITLE
Remove no Movements scenes but Keep those with Audio

### DISCUSSION THREAD

**Discussion by mauamolat** [WyattBlue/auto-editor#387] [1. Help] [answered]
I'd like to be able to remove no movements scenes but keep those with audio.

I found this command: auto-editor "video.mp4" --edit "(or audio:4% motion:2%)" but I don't know if it does the job. I believe it removes both, the no movement and silence but also removes no movement with audio. Do you have a way for my purpose to do on the tool?

I will use it for my videos that I don't speak too much but have movements in it. Thank you!

**Comment by WyattBlue** [comment, marked as answer]
`--edit '(or audio:4% motion:2%)'` does remove motion unless there's audio. You're confusing `or` with `and`

**Reply by mauamolat** [reply]
So you mean this command will do EXACTLY for my purpose? Oh... What values do you recommend? Because when I tried, it cuts off all my audios too.

Thank you, Mate!!! Where can we donate for your project? I'll link it on my YouTube video
